pages/status.tsx

```diff
@@ -26,6 +27,21 @@ const initialValues: CheckStatusRequestBody = {
 const Status: FC = () => {
   const { t } = useTranslation('status')
 
+  const noMatchHref: LinkSummaryItem[] = [
+    {
+      href: 'https://ircc.canada.ca/english/passport/map/map.asp',
+      text: t('no-match-status-links.near-you'),
+    },
+    {
+      href: 'https://www.canada.ca/en/immigration-refugees-citizenship/services/canadian-passports/contact-passport-program.html',
+      text: t('no-match-status-links.contact-program'),
+    },
+    {
+      href: 'https://eservices.canada.ca/en/reservation/',
+      text: t('no-match-status-links.appointment'),
+    },
+  ]
```

Comment:
I think you could have the whole object come from the translation. The links need to be translated too (french links for the french page)